hyrax.gemspec

```diff
@@ -51,19 +51,19 @@ EOF
   spec.add_dependency 'legato', '~> 0.3'
   spec.add_dependency 'posix-spawn'
   spec.add_dependency 'jquery-ui-rails', '~> 5.0'
-  spec.add_dependency 'redis-namespace', '~> 1.5.2'
-  spec.add_dependency 'flot-rails', '~> 0.0.6'
+  spec.add_dependency 'redis-namespace', '~> 1.5'
+  spec.add_dependency 'flot-rails', '~> 0.0'
```

Comment:
I'd say with a 0.0.x gem, it's very important to pin to the minor version (as it's not stable until 1.0 per semver)